cmd/snap-confine/mount-support.c

```diff
@@ -421,9 +421,8 @@ static void sc_bootstrap_mount_namespace(const struct sc_mount_config *config)
 	}
 	// Create the hostfs directory if one is missing. This directory is a part
 	// of packaging now so perhaps this code can be removed later.
-	if (access(SC_HOSTFS_DIR, F_OK) != 0) {
-		debug("creating missing hostfs directory");
-		if (mkdir(SC_HOSTFS_DIR, 0755) != 0) {
+	if (mkdir(SC_HOSTFS_DIR, 0755) < 0) {
+		if (errno != EEXIST) {
 			die("cannot perform operation: mkdir %s",
```

Comment:
BTW. there was a way to create a model of a function that lets coverity know that given call aborts/panics.

Reply:
Yeah, I know about that. But I believe our annotation on die() is sufficient for covert to know this. I can experiment and explicitly add the // [coverity] annotation for it. We'll know after a few more scans (and hopefully once we hook up the autoscanner)